docs/self-managed/zeebe-deployment/configuration/broker.md

```diff
@@ -219,17 +219,23 @@ backup:
+:::note
+
+You can use any S3 compatible storage, including, but not limited to, Amazon S3.
+
+:::
+
+| Field                | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           | Example Value                                 |
+| -------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------- |
+| bucketName           | Name of the bucket where the backup will be stored. The bucket must be already created. The bucket must not be shared with other zeebe clusters. bucketName must not be empty. This setting can also be overridden using the environment variable `ZEEBE_BROKER_DATA_BACKUP_S3_BUCKETNAME`.                                                                                                                                                                                                                                                           |                                               |
+| endpoint             | Configure URL endpoint for the store. If no endpoint is provided, it will be determined based on the configured region. This setting can also be overridden using the environment variable `ZEEBE_BROKER_DATA_BACKUP_S3_ENDPOINT`.                                                                                                                                                                                                                                                                                                                    | https://minio.default.svc.cluster.local:9000/ |
```

Comment:
I added the MinIO example as https.

Reply:
@akeller thanks! Maybe worth noting: The https://minio.default.svc.cluster.local:9000/ url is not very generic, it's the result of one particular way to deploy MinIO. The endpoint can be any arbitrary URL, as long as there's an API behind that that speaks S3.

@pierre-yves-monnet Let's keep this focused on S3, the other providers are a separate discussion.
> The idea is not to give the documentation of the provider but to explain what Zeebe expects for each parameter and link to the provider documentation

As far as I know, there is no such documentation for either MinIO or AWS S3. If you think otherwise, please link to it.

> For example, it's necessary to know that Zeebe waits for an access/Secret Key when he needs to call MinIo, because this is part of the Zeebe Code

This is not correct, Zeebe makes no distinction between MinIO and AWS S3. The auto-discovery features of the AWS S3 client that we use are documented here: https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/credentials-chain.html and can be used to configure the client for MinIO too.